wordpress-mobile/WordPress-Android
# Aztec - Fix problems with media thumbnails that were loaded at different sizes

Fixes #6945 (and partially #6942 that requires an Aztec patch for full resolution), by making sure pictures added to the editor are properly resized by the MediaLoader(s) to the correct width requested by the editor. The max width for pictures is set by us at run-time by using the value returned in `ImageUtils.getMaximumThumbnailWidthForEditor(getActivity())`.

One of the problem fixed was different thumb sizes for the same picture in the editor. The culprit  is the way the preview thumb is created by the functions that load it from the disk, or from the network. Volley code does only take in consideration the width of the picture, instead our ImageUtils does take in consideration both height and width of the picture. So the different sizes of the preview thumb for portrait pictures if loaded from the network.

Also fixed the problem with placeholders that could appear too small in some cases. 
_Note that for a full resolution of this, error placeholder that appear when a network error occurs,  we need a patch in Aztec._

Test 1:
- Add pictures to the editor. 
- Wait until both of those are uploaded
- Exit the editor
- Re-open the editor and pictures should have the same size

Test 2:
- Add pictures to the editor from local device. 
- Add picture to the editor from Media Library
- All the pictures should have the same size (* make sure to use pictures bigger than the max size, since pictures are not upscaled. If you use small pictures than their real size is used).

cc  @mzorz
